torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -720,7 +720,7 @@ class ShapePropagator {
             "aten::log1p(Tensor self) -> Tensor",
             "aten::log2(Tensor self) -> Tensor",
             "aten::log_sigmoid(Tensor self) -> Tensor",
-            "aten::log_softmax(Tensor self, int dim) -> Tensor",
+            "aten::log_softmax(Tensor self, int dim, *, int? dtype) -> Tensor",
```

Comment:
The comment above these sets of ops require that the scalar type of the input is preserved. Now that dtype is an argument it is not longer valid to have these ops in this set.